course_content/iris_course/3.Subcube_Extraction.ipynb

```diff
@@ -351,7 +351,7 @@
    "source": [
     "To combine different Constraints we use an `&`.\n",
     "\n",
-    "For example, below we extract the data named `air_potential_temperature`, *and* with model level numbers 4 and 10."
+    "For example, below we extract the data named `air_temperature`, *and* with model level numbers 4 and 10."
```

Comment:
Likewise, correct as-is

Reply:
Thanks, have changed back. See [this commit](https://github.com/SciTools/courses/pull/177/commits/bd34c93bfb9945f4d1d90770f6534dcb414645ce)